lib/x86/setup.c

```diff
@@ -3,6 +3,7 @@
  *
```

Comment:
+2

Reply:
Ugh. One thing that doesn't work well about github comments is I tried to leave a `+2` on a specific commit, but instead the comment is just left on the commit in its entirety.

Reply:
Ok, I'm just deleting these +2 comments, since they didn't work how I thought. I've got all of my +2's in a google sheet. I'll share that with you instead (shortly).